context/PokemonContextUtils.tsx

```diff
@@ -0,0 +1,12 @@
+import { Pokemon } from '../api/pokemon';
+
+export enum PokemonActionTypes {
+  SAVE_FAV_POKEMON = 'SAVE_FAV_POKEMON',
+  LOAD_FAV_POKEMON = 'LOAD_FAV_POKEMON'
+}
+
+export interface PokemonStateInterface {
```

Comment:
It's fine, just wanted to point out that some people like to use such approach: `IPokemonState`, which is Java-ish, but I'm not a fan of this approach and your name is okay 👍